packages/calling-stateful-client/src/TypeAssertions.ts

```diff
@@ -28,6 +28,8 @@ type InferredCallErrorTargets =
    * stabilized in @azure/commmunication-calling.
    * TODO: Remove this hack once 'CallAgent.feature' becomes part of stable @azure/communication-calling.
    */
+  /* @conditional-compile-remove(calling-1.5.4-beta.1) */
+  | 'CallClient.getEnvironmentInfo'
```

Comment:
I can't seem to remove this addition to the union type. Could use some help here. @PorterNan

Reply:
able to remove it from stable but now the comment exists in beta :(

Reply:
Did you resolve the issue in the end? Is it just comment left in the .md file or something else?

Reply:
Its only the comment left in the .md file. I am not sure how to remove this one.